bin/cron/deploy_to_levelbuilder

```diff
@@ -4,7 +4,8 @@ abort 'Script already running' unless only_one_running?(__FILE__)
 
 # A script that performs a merge of the test branch into the levelbuilder branch.
 #
-# NOTE: This script assumes it is only run from the test environment.
+# NOTE: This script assumes it is only run from an environment with a git worktree
```

Comment:
We should schedule some follow up work to automate the configuration of the separate git worktree on staging. This would be accomplished with a Chef cookbook, and I’m hesitant to delay this change.

Reply:
Thinking about this more … I hesitate to even create a Jira Issue to automate that because our future with Chef is uncertain. Might be useful to augment the existing comment with a `TODO`.